athena-elasticsearch/athena-elasticsearch.yaml

```diff
@@ -51,6 +51,12 @@ Parameters:
     Description: "List of one or more domain-names and their respective endpoints (including secret credentials) in the format: domain1=endpoint1,domain2=endpoint2,... (e.g. movies=https://${secret-credentials}:www.mymovies.com)."
     Default: ""
     Type: String
+  QueryTimeoutCluster:
+    Description: "timeout period (in seconds) for Cluster-Health queries used in the generation of parallel scans."
+    Type: Number
+  QueryTimeoutSearch:
+    Description: "timeout period (in seconds) for Search queries used in the retrieval of documents from an index."
+    Type: Number
```

Comment:
Please include default values.

Reply:
Set defaults (in seconds):
  QueryTimeoutCluster:
    Default: 10
  QueryTimeoutSearch:
    Default: 1800